app/templates/static/cookies-privacy.html

```diff
@@ -69,7 +69,7 @@ <h2 class="question__title neptune">
                                 </h2>
 
                                 <div class="question__description mars">
-                                    <p>We use small files (known as ‘cookies’) to collect information about how you
+                                    <p>We use small files (known as &lsquo;cookies&rsquo;) to collect information about how you
```

Comment:
We usually just put the quotes in like you had originally, but i'm okay with this too 😄

Reply:
That's what I did first time as the charset is `utf-8` but Ben's comments through me off.